protos/varfish/v1/query_seqvars.proto

```diff
@@ -0,0 +1,247 @@
+syntax = "proto3";
+
+package varfish.v1.seqvars;
+
+// Suggestion from emily that might be worth discussing: a few members where I think the enabled bool can be implied by the presence/nonpresence of an optional submessage  making for cleaner code & less branches
+
+
+
+enum Consequence {
+    // high impact
+    ChromosomeNumberVariation = 0;
+    ExonLossVariant = 1;
+    FrameshiftVariant = 2;
+    RareAminoAcidVariant = 3;
+    SpliceAcceptorVariant = 4;
+    SpliceDonorVariant = 5;
+    StartLost = 6;
+    StopGained = 7;
+    StopLost = 8;
+    TranscriptAblation = 9;
+    // moderate impact
+    ThreePrimeUtrTruncation = 10;
+    FivePrimeUtrTruncaction = 11;
+    ConservativeInframeDeletion = 12;
+    ConservativeInframeInsertion = 13;
+    DisruptiveInframeDeletion = 14;
+    DisruptiveInframeInsertion = 15;
+    MissenseVariant = 16;
+    RegulatoryRegionAblation = 17;
+    SpliceRegionVariant = 18;
+    TbfsAblation = 19;
+    // low impact
+    FivePrimeUtrPrematureStartCodonGainVariant = 20;
+    InitiatorCodonVariant = 21;
+    StartRetained = 22;
+    StopRetainedVariant = 23;
+    SynonymousVariant = 24;
+    // modifier
+    ThreePrimeUtrVariant = 25;
+    FivePrimeUtrVariant = 26;
+    CodingSequenceVariant = 27;
+    ConservedIntergenicVariant = 28;
+    ConservedIntronVariant = 29;
+    DownstreamGeneVariant = 30;
+    ExonVariant = 31;
+    FeatureElongation = 32;
+    FeatureTruncation = 33;
+    GeneVariant = 34;
+    IntergenicVariant = 35;
+    IntronVariant = 36;
+    MatureMirnaVariant = 37;
+    // miRNA
+    Mirna = 38;
+    NmdTranscriptVariant = 39;
+    NonCodingTranscriptExonVariant = 40;
+    NonCodingTranscriptIntronVariant = 41;
+    RegulatoryRegionAmplification = 42;
+    RegulatoryRegionVariant = 43;
+    TfBindingSiteVariant = 44;
+    TfbsAmplification = 45;
+    TranscriptAmplification = 46;
+    TranscriptVariant = 47;
+    UpstreamGeneVariant = 48;
+}
+
+// Enumeration for recessive mode queries.
+enum RecesssiveMode {
+    Recessive = 0;
+    CompoundRecessive = 1;
+}
+
+// Choices for failing quality thresholds on genotypes.
+enum FailChoice {
```

Comment:
Can be removed

Reply:
There's a fair bit of code referencing this. Should I make all that code treat it as the default of ignore?

Reply:
Default is drop variant.

Reply:
Oki!

Reply:
```
#[serde(rename_all = "kebab-case")]
pub enum FailChoice {
    /// Ignore failure.
    #[default]
    Ignore,
    /// Drop whole variant.
    #[serde(rename = "drop-variant")]
    Drop,
    /// Interpret as "no-call".
    NoCall,
}
```
(for the record in the rust code rn it's set as ignore by default which is why I made that assumption)

Reply:
https://github.com/varfish-org/varfish-server-worker/blob/402d10d98aa9106734db16fcbecd3a193ded24fd/src/seqvars/query/interpreter/quality.rs#L161 how should I handle the (now) failing test cases here now that there's no way to specify the FailChoice?  Should I just remove the ones without drop?

Reply:
(as well as all code referencing other failchoices?)